BLUESPAWN-client/src/util/filesystem/FileSystem.cpp

```diff
@@ -221,62 +208,98 @@ namespace FileSystem{
 		return std::nullopt;
 	}
 
+
 	File::File(IN const std::wstring& path) : hFile{ nullptr }{
-		FilePath = ExpandEnvStringsW(path);

```

Comment:
@Jack-McDowell you keep this line (now line 219), but then make the rest of the function consistently use the path variable instead of the FilePath like it should. I think this is and issue and needs to be fixed?

Reply:
yup